dapps/shop/backend/models/shop_deployment.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
+  const ShopDeployment = sequelize.define(
+    'ShopDeployment',
+    {
+      shopId: DataTypes.INTEGER,
+      ipfsHash: DataTypes.STRING
```

Comment:
These are defined with an underscore in the migration.  Just checking to make sure this has been tested as working?

Reply:
Yes, sequelize wants you to use camel case in the model. So that in the code, when referring to those columns their name is camel case. Sequelize does the translation when it generates the SQL statement.

And yes I tested it (I actually had a bug initially because I had this column as snake case!).